src/openepd/model/org.py

```diff
@@ -111,8 +111,10 @@ class Plant(PlantRef, WithAttachmentsMixin, WithAltIdsMixin):
     pluscode: str | None = pydantic.Field(
         default=None,
         description="(deprecated) Plus code (aka Open Location Code) of plant's location",
-        deprecated="Pluscode field is deprecated. If users need a pluscode they can obtain it from "
-        "`id` like this: `id.spit('.', maxsplit=1)[0]`",
+        json_schema_extra={
+            "deprecated": "Pluscode field is deprecated. If users need a pluscode they can obtain it from "
```

Comment:
https://docs.pydantic.dev/latest/concepts/fields/#deprecated-fields

Reply:
I see, it first introduced in 2.7.

Reply:
Yes, this field does not supported by older version of pydantic